samples/envs/uv128/sh.txt

```diff
@@ -1,9 +1,9 @@
-( 0.785398163397246,  0.785398163397246,  0.785398163397246); // L00, irradiance, pre-scaled base
-( 0.435209733640970, -0.435209733640970,  0.435209733640970); // L1-1, irradiance, pre-scaled base
-( 0.435209733640970,  0.435209733640970, -0.435209733640970); // L10, irradiance, pre-scaled base
-(-0.435209733640970,  0.435209733640970,  0.435209733640970); // L11, irradiance, pre-scaled base
-( 0.000007341786512,  0.000007341786512, -0.000007341786512); // L2-2, irradiance, pre-scaled base
-( 0.000007341786512, -0.000007341786512, -0.000007341786512); // L2-1, irradiance, pre-scaled base
-(-0.000000000000000, -0.000000000000000, -0.000000000000000); // L20, irradiance, pre-scaled base
-(-0.000007341786512,  0.000007341786512, -0.000007341786512); // L21, irradiance, pre-scaled base
-(-0.000000000000001, -0.000000000000001, -0.000000000000001); // L22, irradiance, pre-scaled base
+( 1.666666666666658,  1.249999999999994,  1.249999999999994); // L00, irradiance, pre-scaled base
```

Comment:
Oops, scary :)

Reply:
mh... this is kinda strange. I can't guarantee that they were good before. I'll check visually to see if it looks correct.